### PR TITLE
[INFRA-351] enable ML2/OVN and DVR by default

### PR DESCRIPTION
**Pull request by g0rgamesh**
Dependent on: https://review.opendev.org/c/openstack/kolla-ansible/+/896534
Merged.

### CODE REVIEW

**Comment by markgoddard** [comment]
This branch needs to be rebased onto stackhpc/zed.

File: releasenotes/notes/enable_ovn-b84974f398242dcb.yaml
Comment (markgoddard) [inline releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: If you are using a Neutron plugin other than ML2/OVN, set `kolla_enable_ovn` to `false`.

Comment (markgoddard) [reply releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: Also point out that OVN distributed FIP is enabled, and how to disable it if necessary.

Comment (g0rgamesh) [reply releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]: Done.

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml
Comment (markgoddard) [inline etc/kayobe/environments/ci-builder/stackhpc-ci.yml:26]: Please remove this from the other files I mentioned in the original PR

File: releasenotes/notes/enable_ovn-b84974f398242dcb.yaml
Comment (markgoddard) [inline releasenotes/notes/enable_ovn-b84974f398242dcb.yaml]:
```suggestion
    `neutron_ovn_distributed_fip` to `false` in `etc/kayobe/kolla/globals.yml`.
```

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml
Comment (g0rgamesh) [reply etc/kayobe/environments/ci-builder/stackhpc-ci.yml:26]: Done.

**Review by markgoddard — APPROVED** [review]
LGTM

**Comment by g0rgamesh** [comment]
> LGTM

Thanks, I'll merge once ovs check for Zed is merged upstream.

**Pull request merged**
